### ISSUE TITLE
Ep 04: 'grep' command option combination

### ISSUE THREAD

**Issue by jay058** [swcarpentry/shell-novice#1342]
Hi,

For using multiple options for the 'grep,' you can mention that multiple options can be combined together. For example, in this command
$ grep -n -w -i "the" haiku.txt

you can write as following
$ grep -nwi 'the' haiku.txt

**Comment by ernstki** [comment]
Agreed that this is a good "trick" to learn eventually, not convinced it's vital to beginner-oriented course materials.

I _believe_ that behavior may be specified by POSIX, or at least by common argument-parsing libraries like `getopt`, but enough programs _don't_ behave this way, it may just end up confusing the novice. For example, most X11 client programs, `find`, and many (or most) command-line programs written in Go all prefix their "long" options with a single dash rather than two (GNU-style). 

So it's almost easier to just stick with `grep -n -w -i` or `grep -n -w -v` than it is to explain all the special cases. Recommend leaving as-is.

**Comment by lemythe** [comment]
I think as a novice, it's better to have all options or arguments in one place like $ grep -nwi 'the' haiku.txt instead of $ grep -n -w -i 'the' haiku.txt. This will help you memorize the formed word that each letter separately.

**Comment by gcapes** [comment]
I don't think this is covered anywhere explicitly, but there is an example in episode 3. That would be the place to cover this, *if* we're going to. I usually mention that you can combine single-letter options when there's an example with more than one, but only because I'm so used to doing it

**Comment by bkmgit** [comment]
If added, this could be in an optional call out.
